docs/source/devguide/workflow.rst

```diff
@@ -116,7 +116,7 @@ pip_. From the root directory of the OpenMC repository, run:
 
 .. code-block:: sh
 
-    pip install -e .[test]
+    python -m pip install -e .[test]
```

Comment:
Out of curiosity when did this become the best practice and what exactly does it help with? Does it make sure that OpenMC is installed in the distribution where `python` is available and avoid the chance that the `pip` being used is from another distro?

Reply:
Long discussion of this [here](https://snarky.ca/why-you-should-use-python-m-pip/)